catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceQueryServiceImpl.java

```diff
@@ -248,7 +250,7 @@ public void run() {
 
     workspaceTasks
         .stream()
-        .map(ForkJoinPool.commonPool()::submit)
+        .map(new ForkJoinPool(MAX_THREADS)::submit)
```

Comment:
I found the thing I was looking for yesterday: https://github.com/codice/ddf/blob/7c9cb66f3c9b6f1f54686d848f79544559820fcd/catalog/transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/thread/ForkJoinPoolFactory.java
I think you need to use that class. So we need to move that class to somewhere this code can access it. I had similar issues with  the XML transformers and the security manager.